README.md

```diff
@@ -115,6 +115,8 @@ make up-cumulus
 Since this will avoid deploying the other modules, deployment time will be
 shorter.
 
+One of the outputs from your deployment should be `cumulus_distribution_api_uri`. Copy the value to `.env` for `TF_VAR_cumulus_distribution_url`. Then redeploy.
```

Comment:
For this particular addition, I'm wondering if adding some context here would be useful. I didn't add a cumulus doc link but can do that if it seems fitting